src/main/cpp/commands/arm/ManualArmCommand.cpp

```diff
@@ -21,14 +21,11 @@ void ManualArmCommand::Execute() {
 
    
     double lift_power_ = this->gamepad_->GetRawAxis(1);
-    double up_power_ = this->gamepad_->GetRawAxis(0);
-    double extend_power_ = this->gamepad_->GetRawAxis(2);
+    double extend_power_ = this->gamepad_->GetRawAxis(3) - this->gamepad_->GetRawAxis(2);
```

Comment:
Still confused about this - I thought we did something about this in the Arm PR we just merged?

Reply:
No, we left it as is, but you did comment about it. The way this works is using the triggers on the gamepad (LT and RT), which are axes with a range of [0, 1]. Pressing the left trigger (or maybe the right, I don't have a gamepad handy right now) will set the power to 1, and pressing the other trigger, because it is subtracted, will set it to -1. I could change it to buttons if that would make more sense, or make a separate ManualExtendCommand.

Reply:
I see. This seems like a hack but if it works it works, and we should keep it if the operator/driver likes it. I'll resolve this comment, but it would be good to leave a comment describing what you just explained in the code itself